pressbooks/pressbooks-book
# Remove unused Facebook share code

When users activate the Enable Social Media web theme option in Pressbooks
![Screenshot from 2021-03-17 14-25-58](https://user-images.githubusercontent.com/13485451/111540695-b5013c80-872c-11eb-9e50-2ac0dac1b1d0.png)

we enqueue an unneeded facebook-js file. Unneeded, because we removed the Facebook share button from book home pages several years ago. We discussed this issue in bug scrub and product meeting with @greatislander, who confirmed that this code can be removed from pressbooks-book. This PR removes the Facebook SVG logo and unneeded Facebook-related Javascript from the page head.

To test:
1. Turn on the Enable Social Media sharing option in a book
2. Observe the following scripts in the page head of your book's homepage
![Screenshot from 2021-03-17 14-28-48](https://user-images.githubusercontent.com/13485451/111541048-38229280-872d-11eb-9d9f-ee1ae3e4bacf.png)
3. Pull this PR into your local environment and rebuild JS/stylesheets as needed
4. Observe that the Facebook SVG and Facebook-related scripts are no longer present in the page head
